### ISSUE TITLE
build fails with -DENABLE_TESTS=ON cmake option

### ISSUE THREAD

**Issue by AndrewMeadows** [pocoproject/poco#2876]
I wanted to build the **samples**.  After examining the `CMakeList.txt` file I determined I needed to add the `-DENABLE_TESTS=ON` option to the `cmake` command.  However with that argument supplied the build fails because it tries to copy from a non-existent directory:

```[ 46%] Linking CXX executable ../../bin/Encodings-testrunner
Error copying directory from "/tmp/poco/Encodings/testsuite/data" to "/tmp/poco/cmake-build/Encodings/testsuite/data".
Encodings/testsuite/CMakeFiles/Encodings-testrunner.dir/build.make:116: recipe for target 'bin/Encodings-testrunner' failed
```

As a workaround: manually adding an empty directory in the right place will fix it.

`mkdir ./Encodings/testsuite/data`

**Comment by Bjoe** [comment]
Hi, can you specify which branch/commit id or Release version you use?

**Comment by AndrewMeadows** [comment]
Uh wow.  I must have been smoking crack.  I pulled to the tip of master today and tried again and it built fine with -DENABLE_TESTS=ON (Ubuntu 18.04).  I lost the old commit number but based on some scratchpad files I had copied off to the side whatever I had was from before 2018-06-03 (when the build_cmake.sh script was last updated)!

Very sorry, dunno how I ended up with such an out of date repo.  I'm closing this.
